pkg/util/k8sutil/k8sutil.go

```diff
@@ -96,10 +96,14 @@ func PVCNameFromMember(memberName string) string {
 }
 
 func makeRestoreInitContainers(backupURL *url.URL, token, repo, version string, m *etcdutil.Member) []v1.Container {
+	curlImage := "tutum/curl"
+	if s := os.Getenv("CURL_IMAGE"); len(s) > 0 {
```

Comment:
Why get it from ENV?

Reply:
i was basing this MR on an older version of etcd operator (0.8.2). What is the recommended approach? Do you want this to be a CLI option on the main process?

Reply:
> Do you want this to be a CLI option on the main process?

I'm not sure what you meant by "CLI option"?
This would be better in the EtcdRestore CR spec.

Reply:
I'll reimplement as an option to the EtcdRestore CR spec.